### ISSUE TITLE
Make it a policy to never show Labs when `"show_labs_settings": false` 

### ISSUE THREAD

**Issue by aaronraimist** [vector-im/element-web#22706]
#### What would you like to do?
**Never** show the Labs tab in settings unless `show_labs_settings` is enabled.

#### Why would you like to do it?
It's extremely confusing for users that the Labs tab appears but only has one or two features in it. Many people in `#element-web:matrix.org` have been confused by this.

#### How would you like to achieve it?
If a feature is in beta but we want to allow more people to try it, then it should come out of the Labs tab and be placed elsewhere in settings.

### Have you considered any alternatives?

https://github.com/vector-im/element-web/issues/8498

or

Show all of the Labs settings but have them disabled with a sentence saying these are only available when `show_labs_settings` is enabled.

**Comment by avfigueredo** [comment]
It's a good idea, I've set the show_labs_settings to false, but continues to appear to users, and it would be good whether it doesn't appear.

**Comment by aaronraimist** [comment]
This seems to have regressed. On app.element.io which is running 1.11.21, I see this:

<img width="845" src="https://user-images.githubusercontent.com/5855073/216690928-33edb15b-c20e-455f-9c6e-22b0f0f1c1eb.png">

**Comment by aaronraimist** [comment]
@t3chguy The PR description matches exactly what I wanted but that isn't what got implemented.

Apparently I missed this comment https://github.com/matrix-org/matrix-react-sdk/pull/8969#issuecomment-1281886948 and this commit https://github.com/matrix-org/matrix-react-sdk/pull/8969/commits/3bd9d0bf29e2f224719b31dec06f1f87e8e01a2e
> Keep the "Labs" name in the Settings menu, for the sake of simplicity.

The name of the tab was the major thing I found confusing.

Here's another recent example of someone being confused by it https://matrix.to/#/!YTvKGNlinIzlkMTVRl:matrix.org/$fJbROknwyPsIdyNPAN2R54ifcxtCxY9fv0K1Jq-L7J4?via=matrix.org&via=privacytools.io&via=mozilla.org

**Comment by t3chguy** [comment]
yeah I suggested changing the config name at the time, guess it didn't occur

**Comment by MamasLT** [comment]
Hello!
I will have to agree on this. There is no point in showing Labs, when they are off. ESPECIALLY not the ones that don't even work properly yet, like BETAS.

All Labs page should be GONE, when the settting is "show_labs_settings": false

Only this makes sense.
And then we can discuss, on separate option for enabling beta options (like "show_beta_labs_settings": true/false) , so the homeserver admins would have an option to decide id they want to enable it.
Most of us wont, because clients wont need these features at all.

Thank You.

**Comment by SimonBrandner** [comment]
> yeah I suggested changing the config name at the time, guess it didn't occur

Argh, yeah, we really did forget to change the name...

**Comment by BloodyIron** [comment]
Yeah "show_labs_settings" set to false still shows Labs stuff, even beta, with v1.11.109, this needs to be honoured as a setting please!
